Data-Collection-Inventory/2021Q2_csvs/data_collection_reported_specimen_count.csv

```diff
@@ -0,0 +1,102 @@
+data_collection_reference_id,specimen_type_name,reported_count
+dong_antero,brain section set,46
+dong_retro,brain section set,128
+dong_trio,brain section set,51
+dulac_hyp_sn_10xv2,cell nucleus,148163
+ecker_hu_DNAseq,tissue sample,3
+ecker_hu_sn_ATACseq,cell nucleus,429292
+ecker_hu_sn_mCseq,cell nucleus,39561
+ecker_sn_ATACseq,cell nucleus,2106854
+ecker_sn_mCseq,cell nucleus,223176
+ecker_sn_mCseq_retro,cell nucleus,49494
+ecker_trio,brain section set,36
+feng_fish,cell in slice,145
+feng_fish,cell in slice,145
+feng_morph,whole cell,45
+feng_sn_10xv3,cell nucleus,344746
+feng_sn_dropseq,cell nucleus,834367
+fischl_lsfm,tissue sample,29
+fischl_mri,brain hemisphere,9
+fischl_oct,brain slice,128
+fischl_stereology,tissue sample,16
+huang_antero,brain section set,49
+huang_fmost,whole brain,164
+huang_fmost_cells,whole cell,467
+huang_input_conn,whole brain,5
+huang_pn_bulk_SSv4,bulk nucleus,19
+huang_pn_sn_10xv2,cell nucleus,106093
+huang_pn_sn_10xv3,cell nucleus,157696
+huang_retro,brain section set,50
+huang_sn_10xv2,cell nucleus,469176
+huang_sn_10xv3,cell nucleus,5477265
+huang_stpt_conn,whole brain,232
+huang_stpt_count,whole brain,58
+human_cortex_SS4_Open_GRU,cell nucleus,4911
+human_cortex_SS4_restricted_GRU,cell nucleus,50688
+human_cortex_SS4_restricted_limited,cell nucleus,208
+kriegstein_sc_10xv2,whole cell,736509
+kriegstein_sc_10xv3,whole cell,259015
+kriegstein_sc_ATACseq,cell nucleus,156334
+kriegstein_sn_10xv2,cell nucleus,1823807
+lein_sn_10xv2,cell nucleus,180766
+lein_sn_10xv3,cell nucleus,1709430
+lein_sn_SSv4,cell nucleus,244983
+nowakowski_evobc,whole cell,20424
+nowakowski_pndev_10xv3,cell nucleus,124580
+osten_stpt,whole brain,393
+regev_cisi,brain slice,1
+tasic_ATACseq,whole cell,13093
+tasic_stpt,whole brain,40
+tilgner_sc_p56,cell nucleus,38311
+yang_morf_cells,whole cell,1957
+yang_MORF_confocal,brain section set,22
+zeng_anderson_sc_10xv2,whole cell,168877
+zeng_expr_char,whole brain,53
+zeng_fmost,whole brain,141
+zeng_fmost_cells,whole cell,1698
+zeng_mouselight,whole brain,1
+zeng_retro,whole brain,746
+zeng_sc_10xv2,whole cell,2459753
+zeng_sc_10xv3,whole cell,1315525
+zeng_sc_SSv4,whole cell,156374
+zeng_sn_10xv2,cell nucleus,90902
+zeng_sn_10xv3,cell nucleus,90382
+zeng_sn_SSv4,cell nucleus,6816
+zeng_tolias_pseq_e,cell in slice,1320
+zeng_tolias_pseq_m,cell in slice,1320
+zeng_tolias_pseq_t,cell nucleus,1320
+zhangK_cj_snareseq_AC,cell nucleus,10780
+zhangK_cj_snareseq_R,cell nucleus,12232
+zhangK_hs_snareseq_AC,cell nucleus,159166
+zhangK_hs_snareseq_R,cell nucleus,170224
+zhangl_anatomy,spinal cord slice,11
```

Comment:
not "slice", this is whole spinal cord

Reply:
Changed `zhangl_anatomy` specimen type to (whole) `spinal cord`.